test/e2e/utils/namespace/namespace.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package namespace
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"sigs.k8s.io/node-feature-discovery/pkg/utils"
+)
+
+const (
+	AddOperation    = "add"
+	RemoveOperation = "remove"
```

Comment:
I think it would make sense to specify these in `pkg/utils/jsonpatch.go`

Again, this could be a separate PR converting all uses of `utils.NewJsonPatch` to use these definitions.

WDYT?

Also, bike-shedding on the naming, for the generic pkg/utils version:

```suggestion
	JSONPatchOpAdd    = "add"
	JSONPatchOpRemove = "remove"
```

Reply:
ping @TessaIO

Reply:
I'll change this ASAP